Makefile

```diff
@@ -39,6 +39,9 @@ k8sprovider::
 	(cd provider && VERSION=${VERSION} go generate cmd/${PROVIDER}/main.go)
 	(cd provider && go build -a -o $(WORKING_DIR)/bin/${PROVIDER} -ldflags "-X ${PROJECT}/${VERSION_PATH}=${VERSION}" $(PROJECT)/provider/v2/cmd/$(PROVIDER))
 
+k8sprovider_debug::
+	(cd provider && go build -a -o $(WORKING_DIR)/bin/${PROVIDER} -gcflags="all=-N -l" -ldflags "-X ${PROJECT}/${VERSION_PATH}=${VERSION}" $(PROJECT)/provider/v2/cmd/$(PROVIDER))
```

Comment:
Looks fine! But for my knowledge, what do the `-N -l` flags do?

Reply:
Those flags keep the compiler from stripping debug symbols, which is needed for dlv debugging.